### ISSUE TITLE
Importing an album by directory orders songs wrong

### ISSUE THREAD

**Issue by SpecialBomb** [DeaDBeeF-Player/deadbeef#1961]
### Steps to reproduce the problem
1. Import an album by folder (filenames must not contain track number at the beginning).

### What's going on? Describe the problem in as much detail as possible.
What I believe is happening is that when deadbeef imports a folder as a playlist, it orders it by filename. For some reason, deadbeef doesn't hold the track number tag in as high as a regard, which causes the player to ignore them by default.

### Information about the software:
I have not done anything special to deadbeef for this to happen.

Deadbeef version: 0.7.2
OS: Debian Testing

### How I would solve this:
When deadbeef imports by folder, it will first check if there is a track number in the file tags. Only sort the tracks this way if the album tag is the same. When doing a recursive add with multiple directories, order the tracks in groups by album name. If there are no track number tags, then sort by filename.

I would also add some kind of "default imported files sort" option in preferences, which would help avoid this altogether. Deadbeef does _**not**_ save your sorting method on restart, so it always defaults by filename.

**Comment by SpecialBomb** [comment]
@Alexey-Yakovenko Well, could you at least think about the imported files filter option for people that would like their imported folders to be sorted in the way they want?

**Comment by Alexey-Yakovenko** [comment]
is that any different from the topics discussed in #1560?

**Comment by SpecialBomb** [comment]
No, that post processing script feature is pretty much exactly what I am thinking of. Do you have any plans on adding that in at all?

**Comment by Alexey-Yakovenko** [comment]
I'm working on a completely different thing atm, but I'm pretty sure I'll have to add this sorting algorithm when I resume working on the media library.
